projects/telemetry/inc/nmea.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+// This module is used to parse and represent NMEA data. Used in the GPS module
+
+#include <stdint.h>
+#include <stdio.h>
+#include "status.h"
+
+// This enum contains the list of NMEA sentences (not all of then are supported yet)
+// because they do not contain useful information
+typedef enum {
+  NMEA_MESSAGE_ID_UNKNOWN = 0,
+  NMEA_MESSAGE_ID_GGA,
+  NMEA_MESSAGE_ID_GLL,
+  NMEA_MESSAGE_ID_GSA,
+  NMEA_MESSAGE_ID_GSV,
+  NMEA_MESSAGE_ID_RMC,
+  NMEA_MESSAGE_ID_VTG,
+  NUM_NMEA_MESSAGE_ID_TYPES
+} NmeaMessageId;
+
+// An enum to represent a position fix's validity (defined by 1, 2, or 6 as described in
+// https://www.linxtechnologies.com/wp/wp-content/uploads/rxm-gps-f4.pdf
+// on page 14)
+
+typedef enum {
+  NMEA_PF_INVALID = 0,
+  NMEA_PF_GPS_SPS = 1,
+  NMEA_PF_DIFFERENTIAL_GPS_SPS = 2,
+  NMEA_PF_DEAD_RECKONING_MODE = 6,
+  NUM_NMEA_POSITION_FIX_TYPES = 4
```

Comment:
Should be `NUM_NMEA_POSITION_FIXES`

Reply:
Makes sense, made the change

Reply:
The alternative is to rename your thing to `POSITION_FIX_TYPE`.

Reply:
I'll stick to the first one, slightly less typing 😄